.github/workflows/dcr-setup.yml

```diff
@@ -0,0 +1,17 @@
+name: DCR Setup
+
+on:
+  - workflow_call
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install Node
```

Comment:
The idea here is to seed our dependencies cache before all of the other jobs start trying to access it, this might not really be necessary.